test/Plutus/Time.purs

```diff
@@ -93,8 +93,8 @@ mkEpochLength = EpochLength <<< BigInt.fromInt
 mkSlotLength :: Int -> SlotLength
 mkSlotLength = SlotLength <<< BigInt.fromInt
 
-mkSafeZone :: Int -> SafeZone
-mkSafeZone = SafeZone <<< BigInt.fromInt
+mkSafeZone :: Int -> Maybe SafeZone
+mkSafeZone = Just <<< SafeZone <<< BigInt.fromInt
```

Comment:
Why `Maybe`?

Reply:
Ogmios says safeZone can be null/nothing. To accept fixtures which used this I had to make it Maybe. Idk if there's a difference between 0 and Nothing

Reply:
I mean why `Maybe` in this function, it can be just `Int -> SafeZone` as no checks are present

Reply:
It's so it's convenient to build `EraSummaryParameters`, as that now has its `safeZone` field as a `Maybe`

Reply:
We should definitely wrap it into maybe at the caller side.